lib/index.js

```diff
@@ -101,7 +137,7 @@ export default class HyperList {
   refresh (element, userProvidedConfig) {
     Object.assign(this._config, defaultConfig, userProvidedConfig)
 
-    if (!element || element.nodeType !== 1) {
+    if (!element || (typeof element.render !== 'function' && element.nodeType !== 1)) {
```

Comment:
I suppose any element of a VirtualDOM has a `render` function, cf. [Matt-Esch/virtual-dom](https://github.com/Matt-Esch/virtual-dom)